manager/src/main/java/com/redhat/service/bridge/manager/models/Processor.java

```diff
@@ -29,7 +32,14 @@
         @NamedQuery(name = "PROCESSOR.findByBridgeIdAndName",
                 query = "from Processor p where p.name=:name and p.bridge.id=:bridgeId"),
         @NamedQuery(name = "PROCESSOR.findByStatusesAndShardId",
-                query = "from Processor p join fetch p.bridge where p.status in (:statuses) and p.bridge.status='AVAILABLE' and p.shardId=:shardId"),
+                query = "select p " +
```

Comment:
@lucamolteni I just want to check my memory of our discussions: We said that we would make this more generic in the future so that we can more easily determine if all resources for either a BridgeInstance or a Processor are ready. This is the first iteration of this.

Reply:
As this method is getting more and more specific, I think we should consider it in the future. I'm closing it for now